dispatch/dispatch.go

```diff
@@ -133,7 +138,17 @@ func (d *Dispatcher) Groups(matchers []*labels.Matcher) AlertOverview {
 				if !a.EndsAt.IsZero() && a.EndsAt.Before(now) {
 					continue
 				}
+
 				status := d.marker.Status(a.Fingerprint())
+
+				if !showSilenced && len(status.SilencedBy) != 0 {
+					continue
+				}
+
+				a.Annotations = a.Annotations.Merge(
+					model.LabelSet{"receiver": model.LabelValue(route.RouteOpts.Receiver)},
```

Comment:
I believe due to the `continue` field in the config, an alert can be sent by multiple receivers, this value should be a list.

It makes me question whether we should be doing it this way, or maybe go the other way and have an index of the receivers to the leaf routes/aggregation groups and filter the alerts by the series of matchers to that leaf. This makes selecting multiple receivers more difficult though.